src/gcp_scanner/crawler/app_services_crawler.py

```diff
@@ -39,7 +39,7 @@ def crawl(self, project_name: str, service: discovery.Resource) -> Dict[str, Any
     app_services = dict()
     try:
       request = service.apps().get(appsId=project_name)
-      response = request.execute()
+      response = await request.execute()
```

Comment:
@mshudrak This is the exact line of code where we need to await and make it asynchronous.

The problem that you got is similar to that of mine, i.e., the object (can be a list or dictionaries, since that is what we primarily return) can't await.